src/alire/alire-warnings.ads

```diff
@@ -0,0 +1,15 @@
+package Alire.Warnings with Preelaborate is
+
+   --  TODO: thread-unsafe
+
+   procedure Warn_Once (Text : String;
+                        ID   : String := "";
```

Comment:
Maybe the ID could be an enum here.

Reply:
The drawback I see with that is that is that it doesn't allow to use the Text instead of the id. For example, if the warning contains some custom info, we may want to repeat it, but not for the same info.

Reply:
A middle ground is by convention define any Ids in that same spec, so they're more visible and documented. I'll try it with the one we have now.